hudi-utilities/src/test/java/org/apache/hudi/utilities/functional/TestHoodieMultiTableDeltaStreamer.java

```diff
@@ -38,16 +38,14 @@
 import java.io.IOException;
 import java.util.Arrays;
 import java.util.List;
-import java.util.Random;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertThrows;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
-public class TestHoodieMultiTableDeltaStreamer extends TestHoodieDeltaStreamer {
+public class TestHoodieMultiTableDeltaStreamer extends TestHoodieDeltaStreamerBase {
```

Comment:
Previously TestHoodieMultiTableDeltaStreamer extended from TestHoodieDeltaStreamer and so tests in TestHoodieDeltaStreamer were running twice. This refactoring will ensure that TestHoodieDeltaStreamer tests run only once.

Reply:
Nice catch ~